src/gui/AstroCalcDialog.cpp

```diff
@@ -5102,56 +5102,57 @@ void AstroCalcDialog::populateWutGroups()
-	wutCategories.insert(q_("Sednoids"), 13);
-	wutCategories.insert(q_("Planetary nebulae"), 14);
-	wutCategories.insert(q_("Bright double stars"), 15);
-	wutCategories.insert(q_("Bright variable stars"), 16);
-	wutCategories.insert(q_("Bright stars with high proper motion"), 17);
-	wutCategories.insert(q_("Symbiotic stars"), 18);
-	wutCategories.insert(q_("Emission-line stars"), 19);
-	wutCategories.insert(q_("Supernova candidates"), 20);
-	wutCategories.insert(q_("Supernova remnant candidates"), 21);
-	wutCategories.insert(q_("Supernova remnants"), 22);
-	wutCategories.insert(q_("Clusters of galaxies"), 23);
-	wutCategories.insert(q_("Interstellar objects"), 24);
-	wutCategories.insert(q_("Globular star clusters"), 25);
-	wutCategories.insert(q_("Regions of the sky"), 26);
-	wutCategories.insert(q_("Active galaxies"), 27);
+	wutCategories.insert(q_("Planets"), 					wut_planets);
+	wutCategories.insert(q_("Bright stars"), 				wut_bright_stars);
+	wutCategories.insert(q_("Bright nebulae"), 				wut_bright_nebulae);
+	wutCategories.insert(q_("Dark nebulae"), 				wut_dark_nebulae);
+	wutCategories.insert(q_("Galaxies"), 					wut_galaxies);
+	wutCategories.insert(q_("Open star clusters"), 				wut_open_star_clusters);
+	wutCategories.insert(q_("Asteroids"), 					wut_asteroids);
+	wutCategories.insert(q_("Comets"), 					wut_comets);
+	wutCategories.insert(q_("Plutinos"), 					wut_plutinos);
+	wutCategories.insert(q_("Dwarf planets"), 				wut_dwarf_planets);
+	wutCategories.insert(q_("Cubewanos"), 					wut_cubewanos);
+	wutCategories.insert(q_("Scattered disc objects"), 			wut_scattered_disc_objects);
+	wutCategories.insert(q_("Oort cloud objects"), 				wut_Oort_cloud_objects);
+	wutCategories.insert(q_("Sednoids"), 					wut_sednoids);
+	wutCategories.insert(q_("Planetary nebulae"), 				wut_planetary_nebulae);
+	wutCategories.insert(q_("Bright double stars"), 			wut_bright_double_stars);
+	wutCategories.insert(q_("Bright variable stars"), 			wut_bright_variable_stars);
+	wutCategories.insert(q_("Bright stars with high proper motion"),	wut_bright_stars_with_high_proper_motion);
+	wutCategories.insert(q_("Symbiotic stars"), 				wut_symbiotic_stars);
+	wutCategories.insert(q_("Emission-line stars"), 			wut_emission_line_stars);
+	wutCategories.insert(q_("Supernova candidates"), 			wut_supernovae_candidates);
+	wutCategories.insert(q_("Supernova remnant candidates"), 		wut_supernovae_remnant_candidates);
+	wutCategories.insert(q_("Supernova remnants"), 				wut_supernovae_remnants);
+	wutCategories.insert(q_("Clusters of galaxies"), 			wut_clusters_of_galaxies);
+	wutCategories.insert(q_("Interstellar objects"),			wut_interstellar_objects);
+	wutCategories.insert(q_("Globular star clusters"),			wut_globular_star_clusters);
+	wutCategories.insert(q_("Regions of the sky"), 				wut_regions_of_the_sky);
+	wutCategories.insert(q_("Active galaxies"), 				wut_active_galaxies);
+	wutCategories.insert(q_("Algol (Beta Persei)-type eclipsing systems"),	wut_Algol_types);
```

Comment:
Is name `Algol (Beta Persei)-type eclipsing systems` really need to use? Why not use simple name `Algol-type eclipsing systems`?

Reply:
good question. I took this literally from [GCVS](http://www.sai.msu.su/gcvs/gcvs/vartype.htm): 
> EA   Algol (Beta Persei)-type eclipsing systems. Binaries with spherical or slightly ellipsoidal components. ...

Maybe "Algol-type eclipsing systems" ?